_pages/en_US/finalizing-setup.txt

```diff
@@ -131,17 +131,18 @@ If, before following this guide, you already had an EmuNAND setup and would like
   + This process will take some time
   + If you get an error, make sure that you have at least 1.3GB of free space on your SD card
 1. Press (A) to continue
-1. Press (B) to return to the main menu
-1. Select "Exit"
-1. Press (A) to relock write permissions if prompted
-1. Navigate to `[M:] MEMORY VIRTUAL`
-1. Press (A) on `boot9.bin` to select it
-1. Select "Copy to 0:/gm9/out"
+1. Select "Dump Options"
+1. Select "Dump Boot9.bin & Boot11.bin"
+1. Press (A) to confirm
+1. Press (A) to continue
+1. Select "Dump OTP.bin"
```

Comment:
This step is unnecessary as `essential.exefs` includes `OTP.bin` and is dumped much earlier in the guide.
Perhaps consider replacing "Dump OTP.bin" with "Dump essential.exefs" instead, to make sure it is actually dumped? (We do see a handful of helpees in the Discord asking about no exefs) But someone else should check on that first.